todo_app/flask_config.py

```diff
@@ -2,7 +2,16 @@
 
 
 class Config:
-    """Base configuration variables."""
-    SECRET_KEY = os.environ.get('SECRET_KEY')
-    if not SECRET_KEY:
-        raise ValueError("No SECRET_KEY set for Flask application. Did you follow the setup instructions?")
+    def __init__(self):
+        """Base configuration variables."""
+        SECRET_KEY = os.environ.get('SECRET_KEY')
+        #API Key and token for Trello
+        TRELLO_API_KEY=os.environ.get('TRELLO_API_KEY')
+        TRELLO_TOKEN=os.environ.get('TRELLO_TOKEN')
```

Comment:
These aren't used... your `TodoService` is what actually reads and uses these env vars.

So delete these lines

Reply:
These are used to raise a ValueError if the config variables are not set